### ISSUE TITLE
User Stories

### ISSUE THREAD

**Issue by FabSugandhi** [FabSugandhi/KasunFabianRyley_T3A2-A#4]
1. User Identification
- Identify who will use the website
- Gather resources and research to understand user needs

2. Create User Personas
- Develop profiles for different types of users based on their needs and goals

3. Write User Stories
- Use first person format
- Make sure each story is easy to understand and specific (including the user type, the websit action they want, and their goal of using the website)

4. Break Down Big Tasks
- Begin with larger goals or features
- Split these into smaller, manageable stories

5. Prioritize
- Decide which stories are most important and should be done first

6. Set Acceptance Criteria
- Write conditions that must be met for the stories to be considered done

7. Track Progress
- Document and track the stories in a project management tool

8. Review and Refine
- Get feedback and adjust stories if needed
- Make sure the implementation meets the acceptance criteria
- Keep updating based on feedback and any new needs
